wdl-lint/src/rules/expression_spacing.rs

```diff
@@ -27,63 +27,63 @@ fn prefix_whitespace(span: Span) -> Diagnostic {
     Diagnostic::note("prefix operators may not contain whitespace")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("remove the internal whitespace")
+        .with_fix("remove the whitespace")
 }
 
 /// Reports missing following whitespace around operators
 fn missing_surrounding_whitespace(span: Span) -> Diagnostic {
     Diagnostic::note("operators must be surrounded by whitespace")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("add a space before and after this operator")
+        .with_fix("add whitespace around this operator")
 }
 
 /// Reports missing preceding whitespace around operators
 fn missing_preceding_whitespace(span: Span) -> Diagnostic {
     Diagnostic::note("operators must be preceded by whitespace")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("add a space before this operator")
+        .with_fix("add whitespace before this operator")
 }
 
 /// Reports missing following whitespace around operators
 fn missing_following_whitespace(span: Span) -> Diagnostic {
     Diagnostic::note("operators must be followed by whitespace")
         .with_rule(ID)
         .with_highlight(span)
-        .with_fix("add a space after this operator")
```

Comment:
I like this one better, as we want them to use a space specifically (not a tab, for instance, which I consider to be "whitespace").